client/src/Components/LoginModal.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Popover } from 'react-bootstrap';
+import { ButtonGroup } from 'react-bootstrap';
+import { Button } from 'react-bootstrap';
+import { Modal } from 'react-bootstrap';
+import { Tooltip } from 'react-bootstrap';
+import { FormGroup } from 'react-bootstrap';
+import { FormControl } from 'react-bootstrap';
+import { ToggleButton } from 'react-bootstrap';
+
+class LoginModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleShow = this.handleShow.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleButtonSubmit = this.handleButtonSubmit.bind(this);
+    console.log('PROPS', this.props)
+    this.state = {
+      show: false,
+      username: '',
+      password: '',
+    };
+  }
+
+  handleClose() {
+    this.setState({ show: false });
+  }
+
+  handleShow() {
+    this.setState({ show: true });
+  }
+
+  handleChange(e) {
+    if(e.target.placeholder === 'Username') {
```

Comment:
Is a FormControl similar to a regular Input tag? If so I'm not sure if `placeholder` is the best property to tell whether it's for passwords or usernames. Maybe `id` or `name`